ui/app/services/stats-trackers-registry.js

```diff
@@ -11,6 +11,9 @@ import AllocationStatsTracker from 'nomad-ui/utils/classes/allocation-stats-trac
 const MAX_STAT_TRACKERS = 10;
 let registry;
 
+const exists = (tracker, prop) =>
+  tracker.get(prop) && !tracker.get(prop).isDestroyed && !tracker.get(prop).isDestroying;
```

Comment:
this could eventually be a util 👍

Reply:
It could, but ideally I never have to use something like this again 😅